setup.py

```diff
@@ -39,7 +39,6 @@ def run(self):
 install_requires = [
     "ply>=3.9",  # See https://github.com/pyinstaller/pyinstaller/issues/1945
     "configparser>=3.5.0",
-    "zc.lockfile>=1.2.1",
```

Comment:
Will need to remove it from https://github.com/conda-forge/dvc-feedstock/blob/master/recipe/meta.yaml as well

Reply:
@Suor Could you send a PR for that please?

Reply:
Need a dvc release that uses new locks first.